lnrpc/walletrpc/psbt.go

```diff
@@ -47,8 +47,11 @@ func verifyInputsUnspent(inputs []*wire.TxIn, utxos []*lnwallet.Utxo) error {
 
 // lockInputs requests a lock lease for all inputs specified in a PSBT packet
 // by using the internal, static lock ID of lnd's wallet.
-func lockInputs(w lnwallet.WalletController, packet *psbt.Packet) (
+func lockInputs(w lnwallet.WalletController, packet *psbt.Packet, lockDuration time.Duration) (
```

Comment:
nit: line length. And after a multi-line function declaration, there should be an empty line.